2.6/root/usr/share/container-scripts/mongodb/replset-functions.sh

```diff
@@ -0,0 +1,97 @@
+# This file contains functions	 for replSet manipulation
+
+# cache_container_addr waits till the container gets the external IP address and
+# cache it to disk
+function cache_container_addr() {
+  echo -n "=> Waiting for container IP address ..."
+  local i
+  for i in $(seq ${MAX_ATTEMPTS}); do
+    if ip -oneline -4 addr show up scope global | grep -Eo '[0-9]{,3}(\.[0-9]{,3}){3}' > "${HOME}"/.address; then
```

Comment:
It is better to use only one IP for connection via mongo shell - I think `mongo --host IP1 IP2` could cause an error. Thanks.

This is also present in master branch.